tests/databases/api_tests.py

```diff
@@ -228,6 +228,7 @@ def test_create_database(self):
         database_data = {
             "database_name": "test-create-database",
             "sqlalchemy_uri": example_db.sqlalchemy_uri_decrypted,
+            "configuration_method": ConfigurationMethod.SQLALCHEMY_URI,
```

Comment:
Can you add an assert below in this test verifying that the attribute was set correctly?

For example, between lines 241-242:

```python
assert model.configuration_method == ConfigurationMethod.SQLALCHEMY_URI
```

Reply:
yeah that sounds like a good idea. I will add it.